.gitignore

```diff
@@ -1,22 +1,26 @@
-# Compiled class file
 *.class
 
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
 # Package Files #
 *.jar
 *.war
 *.ear
-*.zip
-*.tar.gz
-*.rar
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# any packs that would be included
+*.exe
+*.tar.gz
+*.zip
+
+# IntelliJ IDEA
+.idea/
+*.iml
+
+rat.txt
+target/
```

Comment:
What would be the reasons for excluding .exe, rat.txt, and target/ folder?

Reply:
@imesh these files to be ignored are not applicable to this context, will remove these.